.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+# Java Maven CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-java/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/openjdk:8-jdk
+
+    working_directory: ~/repo
+
+    environment:
+      # Customize the JVM maximum heap limit
+      MAVEN_OPTS: -Xmx3200m
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "pom.xml" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+
+      - run: mvn dependency:go-offline
+
+      - save_cache:
+          paths:
+            - ~/.m2
+          key: v1-dependencies-{{ checksum "pom.xml" }}
+
+      - run: mvn package
+
+      - store_artifacts:
+          path: target/Minetrix-0.1.0-SNAPSHOT-jar-with-dependencies.jar
```

Comment:
Can we use wildcards here so that when bumping the version we will not have to change this?

Reply:
It would appear that wildcards can't be used: https://discuss.circleci.com/t/wildcards-in-path-definition-for-uploading-artifacts/18992/3

Reply:
Hm, damn.  Well, fine by me for now then